build.gradle.kts

```diff
@@ -33,6 +33,7 @@ dependencies {
 	testImplementation("io.mockk:mockk:1.12.3")
 	testImplementation("org.amshove.kluent:kluent:1.64")
 	testImplementation("org.mockito:mockito-inline:3.3.3")
+	runtimeOnly("com.h2database:h2")
```

Comment:
This is will force the application to run with `h2`. We want to run the application with a mysql database, and run the integration tests with `h2` database.